index.html

```diff
@@ -752,6 +752,17 @@ <h3>Validate Algorithm</h3>
 as indicated in the `statusMessages` array.
         </li>
       </ol>
+
+      <p>
+When a `statusListCredential` URL is dereferenced, server implementations MAY provide a
+mechanism to dereference the status list as of a particular point in time. If such a
+feature is supported, and if query parameters are supported by the URL scheme,
+then the name of the query parameter MUST be `timestamp` and the value MUST be
+a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The resulting
+status list credential SHOULD be the status list as it existed at the given
```

Comment:
```suggestion
status list credential MUST be the status list as it existed at the given point in time, or an error MUST be returned.
```

Reply:
Done in 19f616071611e71edae45485cf090ed76de8772c.